pandas/core/generic.py

```diff
@@ -7829,7 +7829,7 @@ def groupby(
         group_keys=True,
         squeeze=False,
         observed=False,
-        **kwargs
+        mutated=False,
```

Comment:
this is changing a public API. The reason the kwargs was added was to work around some internal routines calling this. So I wouldn't do this; rather we can work on cleaning how this is called to avoid passing thru here.